macros/scripts/audio.asm

```diff
@@ -104,28 +143,33 @@ unknownmusic0xe2: MACRO
 	db \1 ; unknown
 ENDM
 
-	enum togglenoise_cmd ; $e3
-togglenoise: MACRO
-	db togglenoise_cmd
-	db \1 ; id
+	enum toggle_noise_cmd ; $e3
+toggle_noise: MACRO
+	db toggle_noise_cmd
+	db \1 ; drum kit
 ENDM
 
-	enum panning_cmd ; $e4
-panning: MACRO
-	db panning_cmd
-	db \1 ; tracks
+	enum force_stereo_panning_cmd ; $e4
+force_stereo_panning: MACRO
+	db force_stereo_panning_cmd
+	dn %\1\1\1\1, %\2\2\2\2 ; left enable, right enable
```

Comment:
(This applies to `stereo_panning` as well)
If these are booleans, why can't I pass `TRUE` and `FALSE`? Please, at least
provide some sort of error checking, or simply drop the arguments in an `IF`
to see if the assembler evaluates them to boolean true.

Reply:
I admit this macro is ugly now. Do you have a concise idea for how to handle `1/TRUE`, `0/FALSE` without an `IF` and `ELSE` for each nibble?

Reply:
Probably something like `_n1 = !!(\1)`.

Reply:
Maybe even combine that into just `dn -!!(\1), -!!(\2)`. I agree this is super ugly but I guess it'd work?

Reply:
Fixed. We were somehow overlooking the simple `dn %1111 * \1, %1111 * \2` haha

Reply:
While that solution is indeed simple (and I was overlooking it :P), anything other than 0 should be treated as TRUE. Please coerce the value into a real boolean by wrapping it in `!!(\1)`. I know it doesn't really matter in this case, but it's something.

Reply:
Good point. Fixed.
`!!(\1)` doesn't seem to work.
`!!(2)` resolves to `2`, thus `stereo_panning FALSE, 2` was resulting in `0x1e` instead of `0x0f`.
But there is the almost-just-as-good `(1 && \1)`.